src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -193,6 +193,16 @@ BinaryCompactObject<UseWorldtube>::BinaryCompactObject(
       PARSE_ERROR(context,
                   "ObjectA's inner radius must be less than its outer radius.");
     }
+    // The length of the cube surrounding each object is length_inner_cube_
+    // / 2.0, this checks to make sure that the shell is not touching the cube.
+    if (object_a.outer_radius >=
+        (length_inner_cube_ / 2.0) + offset_x_coord_a_) {
```

Comment:
Shouldn't this be `object_a.outer_radius >= length_inner_cube / 2.0 - abs(offset_x_coord_a_)`? Please check my math

Reply:
Currently we only let the cube_scale >= 1.0 so I think either way is fine